### PR TITLE
Fix  categorical_crossentropy docstring

### PR DESCRIPTION
**Pull request by ZhuBaohe**
This PR fixes the docstring of tf.keras.backend.categorical_crossentropy method.

### CODE REVIEW

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Why the change? What is wrong with the existing values?

**Comment by ZhuBaohe** [comment]
@mihaimaruseac 

Since **from_logits=False**,  the tensor  **b** expects to the result of a softmax. 

This modification causes `tf.reduce_sum(b, axis=1)` to generate ` array([1., 1., 1.]`.

**Review by mihaimaruseac — APPROVED** [review]
